kinto/core/storage/memory.py

```diff
@@ -13,7 +13,8 @@
     StorageBase,
     exceptions,
 )
-from kinto.core.utils import COMPARISON, find_nested_value, json
+from kinto.core.utils import COMPARISON, find_nested_value
+import json
```

Comment:
Couldn't you import `json` from `kinto.core.utils` here too?

Reply:
Right now, kinto.core.utils redefines json as simdjson, so no. Currently I have:
simdjson for PostgreSQL, and native json for Memory 

Not sure whether or not a more performant (than native) json library is needed but that's the rationale. Alternatively it's possible simdjson could be used for both storages but the way records are added via memory storage would need to be modified which is a far more invasive change.

Reply:
The usage of `json`  in the memory backend is rather straightforward. What issues would we have if we would use `simdjson` there too?

Reply:
The problem is that the unit test I introduced will raise an exception here (found in core/storage/memory.py):

> obj = json.loads(json.dumps(obj))

The reason is that simdjson cannot deserialize strings representing numbers > 64 bits, as per https://github.com/simdjson/simdjson/issues/167. On the other hand, it *can* serialize a > 64-bit number.

Reply:
Ok, gotcha.

And then I would suggest that once we merge this, you create an issue to keep track of this so that we can use simdjson everywhere once this is fixed upstream.

(we could also do something like `
Then what do you think of something like `obj = simdjson.loads(json.dumps(obj))` but I don't think it brings a lot of value

Reply:
Sure thing. I'll create an issue once it's merged and I've subscribed to the simdjson issue to track progress there.